css/style.css

```diff
@@ -0,0 +1,670 @@
+:root {
+  --main-bc: #f5f5f5;
+  --dark-bc: #272a33;
+  --greyish-color: #0a0a0a;
+  --header-grey-color: #666;
+  --greyish-color-90: #a0a0a039;
+  --orange-color: #ec5242;
+  --darkgrey-color: #272a31;
+  --border-color-orange: #ff4f38;
+  --border-color-grey: #e7e7e7;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: var(--main-bc);
+}
+
+/* Fonts Styles */
+
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+.default-family {
+  font-family: "Lato", sans-serif;
+}
+
+.fw-300 {
+  font-weight: 300;
+}
+
+.fw-500 {
+  font-weight: 500;
+}
+
+.fw-600 {
+  font-weight: 600;
+}
+
+.fw-700 {
+  font-weight: 300;
+}
+
+.fw-900 {
+  font-weight: 1200;
+}
+
+.fs-12 {
+  font-size: 12px;
+}
+
+.fs-14 {
+  font-size: 14px;
+}
+
+.fs-16 {
+  font-size: 16px;
+}
+
+.fs-18 {
+  font-size: 18px;
+}
+
+.fs-24 {
+  font-size: 24px;
+}
+
+.fs-36 {
+  font-size: 36px;
+}
+
+.fs-48 {
+  font-size: 75px;
+}
+
+.fs-60 {
+  font-size: 60px;
+}
+
+.fc-greyish {
+  color: var(--greyish-color);
+}
+
+.fc-dark-greyish {
+  color: var(--header-grey-color);
+}
+
+.fc-orange {
+  color: var(--orange-color);
+}
+
+.fc-darkgrey {
+  color: var(--darkgrey-color);
+}
+
+.fc-white {
+  color: #fff;
+}
+
+/* background */
+
+.bc-white {
+  background-color: #fff;
+}
+
+.bc-gray {
+  background-color: var(--main-bc);
+}
+
+.bc-dark {
+  background-color: var(--dark-bc);
+}
+
+/* Margin Pre-set */
+.mb-1 {
+  margin-bottom: 1.2rem;
+}
+
+.mb-2 {
+  margin-bottom: 2rem;
+}
+
+.mr-1 {
+  margin-right: 1rem;
+}
+
+.mr-2 {
+  margin-right: 2rem;
+}
+
+.mt-1 {
+  margin-top: 1rem;
+}
+
+.mt-2 {
+  margin-top: 2rem;
+}
+
+.mt-05 {
+  margin-top: 0.5rem;
+}
+
+/* Padding Pre-set */
+.pl-1 {
+  padding-left: 1rem;
+}
+
+.pt-1 {
+  padding-top: 1rem;
+}
```

Comment:
[OPTIONAL] I suggest that you use new classes only for more than one property because this way you are flooding your html tags with a lot of class names and adding 3 lines for one property , this approach is good when we have a set of properties that we apply to many components